auth_pysaml_website/controllers/main.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2020 GlodoUK <https://www.glodo.uk/>
+
+from odoo.http import request
+
+from odoo.addons.web.controllers.main import Home
+
+
+class SAMLLogin(Home):
+    def _list_saml_providers_domain(self):
+        website_id = getattr(request, "website", None)
+        if website_id:
```

Comment:
I think you should define a base domain and, if a website is available, extend it by adding the below terms.